src/EnergyPlus/EcoRoofManager.cc

```diff
@@ -413,19 +414,9 @@ namespace EcoRoofManager {
                 Gammah = std::pow(1.0 - 5.0 * Rib, -0.5);
             }
 
-            if (RoughSurf == DataSurfaces::SurfaceRoughness::VerySmooth) {
-                state.dataEcoRoofMgr->Zog = 0.0008;
-            } else if (RoughSurf == DataSurfaces::SurfaceRoughness::Smooth) {
-                state.dataEcoRoofMgr->Zog = 0.0010;
-            } else if (RoughSurf == DataSurfaces::SurfaceRoughness::MediumSmooth) {
-                state.dataEcoRoofMgr->Zog = 0.0015;
-            } else if (RoughSurf == DataSurfaces::SurfaceRoughness::MediumRough) {
-                state.dataEcoRoofMgr->Zog = 0.0020;
-            } else if (RoughSurf == DataSurfaces::SurfaceRoughness::Rough) {
-                state.dataEcoRoofMgr->Zog = 0.0030;
-            } else { // VeryRough
-                state.dataEcoRoofMgr->Zog = 0.005;
-            } // TODO: fix this after creating FindEnumeratedValueIndex()
+            // "VeryRough, Rough, MediumRough, MediumSmooth, Smooth, VerySmooth"
+            std::array<Real64, 6> zogLookup = {0.005, 0.0030, 0.0020, 0.0015, 0.0010, 0.0008};
```

Comment:
Small comment. Replace `6` with `static_cast<int>(Material::SurfaceRoughness::Num)`.

Reply:
Right right, for sure